core/src/avm2/globals/XML.as

```diff
@@ -44,6 +45,19 @@ package {
         AS3 native function text():XMLList;
         AS3 native function toString():String;
 
+        AS3 function hasComplexContent():Boolean {
+            return !this.hasSimpleContent();
+        }
+
+        prototype.hasComplexContent = function():Boolean {
+            return !this.hasSimpleContent();
```

Comment:
I think it might be better if this uses an `AS3::` for consistency, but if it works this way, it's fine.